librocksdb_sys/src/generated.rs

```diff
@@ -25,79 +25,122 @@ pub enum DBStatisticsTickerType {
     BlockCacheBytesRead = 18,
     BlockCacheBytesWrite = 19,
     BloomFilterUseful = 20,
-    PersistentCacheHit = 21,
-    PersistentCacheMiss = 22,
-    SimBlockCacheHit = 23,
-    SimBlockCacheMiss = 24,
-    MemtableHit = 25,
-    MemtableMiss = 26,
-    GetHitL0 = 27,
-    GetHitL1 = 28,
-    GetHitL2AndUp = 29,
-    CompactionKeyDropNewerEntry = 30,
-    CompactionKeyDropObsolete = 31,
-    CompactionKeyDropRangeDel = 32,
-    CompactionKeyDropUser = 33,
-    CompactionRangeDelDropObsolete = 34,
-    CompactionOptimizedDelDropObsolete = 35,
-    NumberKeysWritten = 36,
-    NumberKeysRead = 37,
-    NumberKeysUpdated = 38,
-    BytesWritten = 39,
-    BytesRead = 40,
-    NumberDbSeek = 41,
-    NumberDbNext = 42,
-    NumberDbPrev = 43,
-    NumberDbSeekFound = 44,
-    NumberDbNextFound = 45,
-    NumberDbPrevFound = 46,
-    IterBytesRead = 47,
-    NoFileCloses = 48,
-    NoFileOpens = 49,
-    NoFileErrors = 50,
-    StallL0SlowdownMicros = 51,
-    StallMemtableCompactionMicros = 52,
-    StallL0NumFilesMicros = 53,
-    StallMicros = 54,
-    DbMutexWaitMicros = 55,
-    RateLimitDelayMillis = 56,
-    NoIterators = 57,
-    NumberMultigetCalls = 58,
-    NumberMultigetKeysRead = 59,
-    NumberMultigetBytesRead = 60,
-    NumberFilteredDeletes = 61,
-    NumberMergeFailures = 62,
-    BloomFilterPrefixChecked = 63,
-    BloomFilterPrefixUseful = 64,
-    NumberOfReseeksInIteration = 65,
-    GetUpdatesSinceCalls = 66,
-    BlockCacheCompressedMiss = 67,
-    BlockCacheCompressedHit = 68,
-    BlockCacheCompressedAdd = 69,
-    BlockCacheCompressedAddFailures = 70,
-    WalFileSynced = 71,
-    WalFileBytes = 72,
-    WriteDoneBySelf = 73,
-    WriteDoneByOther = 74,
-    WriteTimedout = 75,
-    WriteWithWal = 76,
-    CompactReadBytes = 77,
-    CompactWriteBytes = 78,
-    FlushWriteBytes = 79,
-    NumberDirectLoadTableProperties = 80,
-    NumberSuperversionAcquires = 81,
-    NumberSuperversionReleases = 82,
-    NumberSuperversionCleanups = 83,
-    NumberBlockCompressed = 84,
-    NumberBlockDecompressed = 85,
-    NumberBlockNotCompressed = 86,
-    MergeOperationTotalTime = 87,
-    FilterOperationTotalTime = 88,
-    RowCacheHit = 89,
-    RowCacheMiss = 90,
-    ReadAmpEstimateUsefulBytes = 91,
-    ReadAmpTotalReadBytes = 92,
-    NumberRateLimiterDrains = 93,
+    BloomFilterFullPositive = 21,
```

Comment:
These changes are updated by a script to make the enums consistent with the new version.